lib/connectors/SentinelConnector/index.ts

```diff
@@ -39,6 +41,7 @@ export interface ISentinelConnectionOptions extends ITcpConnectionOptions {
   sentinelPassword?: string;
   sentinels: Array<Partial<ISentinelAddress>>;
   sentinelRetryStrategy?: (retryAttempts: number) => number | void | null;
+  sentinelReconnectStrategy?: (retryAttempts: number) => number | void | null;
```

Comment:
Not sure about this name, could be confusing.

But I'm planning to submit a PR that documents all the sentinel options after this gets merged, so that might help.